Cargo.toml

```diff
@@ -29,8 +29,8 @@ bitcoinconsensus = { version = "0.19.0-1", optional = true }
 serde = { version = "1", optional = true }
 
 [dev-dependencies]
-serde_derive = "<1.0.99"
```

Comment:
I really don't understand these. Under what circumstances would cargo use a different major version than the one that is specified? Does `<1.0.99` really work this way?

Reply:
https://jubianchi.github.io/semver-check/#/%3C1.0.99/0.1.2

```
This constraint does not provide a lower bound which means you will probably get unexpected breaking changes.
```